llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1386,6 +1386,13 @@ def TuneOptimizedZeroStrideLoad
                       "true", "Optimized (perform fewer memory operations)"
                       "zero-stride vector load">;
 
+foreach nf = {2-8} in
+  def TuneOptimizedNF#nf#SegmentLoadStore :
```

Comment:
A random thought: There are 250 features for RISCV now, we may need to increase `MAX_SUBTARGET_FEATURES` soon.

Reply:
Isn't the limit 5*64 or 320?

Reply:
Yes, but that won't last long, especially if we add some fusions (like XiangShan).